examples/rbac/nginx/nginx-ingress-controller-rbac.yml

```diff
@@ -64,14 +64,27 @@ rules:
   - apiGroups:
       - ""
     resources:
-      - configmaps
```

Comment:
Ok. Then I will readd the old `get` for configmap as was present before this PR and just add the new readwrite configmap for the resource and.